src/Jenssegers/Mongodb/Query/Builder.php

```diff
@@ -209,6 +209,15 @@ public function get($columns = [])
         return $this->getFresh($columns);
     }
 
+    /**
+     * @inheritdoc
```

Comment:
@inheritdoc should be replaced with it’s own docblock since it’s not part of Laravel.

Reply:
@jenssegers I updated. Kindly recheck.